stakeholder-mapping/how-we-map.md

```diff
@@ -0,0 +1,26 @@
+## Process
+### Aim
+To have a comprehensive and diverse list of DT stakeholders (e.g. organisational, national, international) that include a) theme-specific stakeholders (e.g. NOC) and b) general stakeholders (DT Hub).
+
+### Method 
+A combination of desk research, targeted questions (e.g. to directors), and exploratory discussions with current stakeholders/partners (e.g. RAM meetings) were undertaken prior to September 2023. These activties identified a first set of stakeholders (n = 92) which were used to build the data collection structure. 
+
+The stakeholder list was developed based on the [RAM sharepoint people list]() and and informal list maintained by the RCM. The descriptors and organisartional units were developed as described above. 
```

Comment:
RAM sharepoint people list misses the link https://thealanturininstitute.sharepoint.com/sites/TRICDTLeadershipteam/Lists/rampeople/AllItems.aspx?sortField=LinkTitle&isAscending=true&viewid=8603f4e7-e1d1-4ca4-90d0-b5a7b36bb009

Reply:
also spelling:
- organisartional -> organisational